### DISCUSSION TITLE
How to set columns from specific row?

### DISCUSSION THREAD

**Discussion by Folliant** [exceljs/exceljs#2107] [General]
I could use `insertRows` to insert data into specific positions. But `worksheet.columns` doesn't accept position and it always in top. I want to add image before table, and add columns from 8 position. How I could do that?

![image](https://user-images.githubusercontent.com/69875014/183676116-c6e5ad82-eda3-4c09-b10b-8ce6f1b0394c.png)
![image](https://user-images.githubusercontent.com/69875014/183695973-9a5d9f8b-22b0-431a-aa38-dcb2f402588c.png)

**Comment by chepe263** [comment]
fill the worksheet as normal and [insert rows](https://github.com/exceljs/exceljs#insert-rows)

1. Insert one row then [duplicate it](https://github.com/exceljs/exceljs#duplicate-a-row)
2. Insert one row eigth times.
